extensions/wikia/RWEPageHeader/styles/_search.scss

```diff
@@ -0,0 +1,193 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/placeholder';
+
+$background-color-mix: 10%;
+$black: #000;
+
+$active-color: lighten($color-links, 20%);
+$inactive-color: lighten($color-text, 50%);
+@if $is-dark-wiki {
+	$active-color: darken($color-links, 20%);
+	$inactive-color: darken($color-text, 50%);
+}
+
+$background-color: mix(mix($color-buttons, $black, $background-color-mix * 5), $color-page, $background-color-mix);
+
+$search-button-height: 100%;
+$search-button-width: 100%;
+$search-close-button-padding-right: 8px;
+$search-expanded-width: 100%;
+$search-initial-width: 110px;
+$search-form-height: 34px;
+$search-form-margin-top: 7px;
+$search-form-wrapper-initial-width: $search-initial-width + 20px;
+$search-submit-button-padding: 5px 9px;
+$search-suggestion-z-index: 2;
+$search-transition-long: .3s;
+$search-transition-short: 50ms 150ms;
+
+#searchInputWrapperRWE {
+	border-bottom: solid 1px transparent;
+}
+
+#searchInputRWE {
+	background: transparent;
+
+	@include placeholder {
+		color: $color-text;
+		font-size: $wds-typescale-size-base;
+		font-weight: bold;
+		line-height: $wds-line-height;
+	}
+}
+
+#searchFormWrapperRWE {
+	border-top: 0;
+	margin-bottom: 0;
+	width: $search-form-wrapper-initial-width;
+
+	&:hover {
+		.wds-global-navigation__search-label {
```

Comment:
Out of curiosity, is there a reason to keep those prefixes? ( .wds-global-navigation )
Unless we reuse a part of styling, it'd make sense to rename them to avoid unintended side effects.